godtossing/Pos/Ssc/GodTossing/Workers.hs

```diff
@@ -144,7 +144,7 @@ checkNSendOurCert sendActions = do
             ourVssCertificate <- getOurVssCertificate slot
             let contents = MCVssCertificate ourVssCertificate
             sscProcessOurMessage (sscProcessCertificate ourVssCertificate)
-            invReqDataFlowNeighborsTK "ssc" (enqueueMsg sendActions) (MsgMPC OriginSender) ourId contents
+            _ <- invReqDataFlowTK "ssc" (enqueueMsg sendActions) (MsgMPC OriginSender) ourId contents
```

Comment:
`enqueueMsg sendActions` decides where the messages goes, so `invReqDataFlowNeighbors` was redundant and has been removed.